Packet++/src/Packet.cpp

```diff
@@ -733,6 +734,10 @@ Layer* Packet::createFirstLayer(LinkLayerType linkType)
 	{
 		return new SllLayer((uint8_t*)rawData, rawDataLen, this);
 	}
+	else if (linkType == LINKTYPE_LINUX_SLL2)
```

Comment:
it'd be nice to add `Sll2Layer::isDataValid(...)` to check that `rawData` is at least in the size of `sizeof(sll2_header)`

Reply:
done